.github/workflows/code_quality.yml

```diff
@@ -89,12 +89,6 @@ jobs:
             php: "8.2"
             composer: "v2"
             opensearch: "2.5.0"
-
-          - magento: "2.4.6"
```

Comment:
Removing this looks out of scope for a rate limiting change.

Reply:
The test was added in error so will just leave the removal here.